start-ecs

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+#
+# Start script for register-api that assumes environment variables are already provided
```

Comment:
Can we take this comment out as it looks to be a copy and pasted from another project anyway

Reply:
Done.